packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -48,21 +48,23 @@ abstract class ProgressIndicator extends StatefulWidget {
     this.semanticsValue,
   }) : super(key: key);
 
-  /// If non-null, the value of this progress indicator with 0.0 corresponding
-  /// to no progress having been made and 1.0 corresponding to all the progress
-  /// having been made.
+  /// If non-null, the value of this progress indicator.
+  ///
+  /// A value of 0.0 means no progress and 1.0 means that progress is complete.
   ///
   /// If null, this progress indicator is indeterminate, which means the
-  /// indicator displays a predetermined animation that does not indicator how
+  /// indicator displays a predetermined animation that does not indicate how
   /// much actual progress is being made.
   final double value;
 
-  /// The progress indicator's background color. The current theme's
-  /// [ThemeData.backgroundColor] by default.
+  /// The progress indicator's background color.
+  ///
+  /// The current theme's [ThemeData.backgroundColor] by default.
   final Color backgroundColor;
 
-  /// The indicator's color is the animation's value. To specify a constant
-  /// color use: `new AlwaysStoppedAnimation<Color>(color)`.
+  /// The indicator's color as an animated value.
```

Comment:
This should either be "progress indicator's", or the previous one should be "indicator's" for consistency.

Reply:
Sounds good, done.